### ISSUE TITLE
Manage Volunteer Opportunities forms break in modal display due to jQuery-ui conflict with Bootstrap

### ISSUE THREAD

**Issue by fulltrucker** [civicrm/org.civicrm.volunteer#485]
When using a theme that loads a version of Bootstrap CSS, back office forms that load in a modal window (such as Define Volunteer Opportunities, Assign Volunteers, View Volunteer Roster, and Log Hours) display incorrectly. The form layout is shifted to the left 50% the width of the modal. Example:

![screen shot 2018-12-04 at 4 03 59 pm](https://user-images.githubusercontent.com/6656507/49484146-30a47d00-f7f3-11e8-9692-c0f827b50ca3.png)

The div ID `crm-volunteer-dialog` has the following classes assigned to it: `ui-dialog-content ui-widget-content modal-dialog`.  Turns out `jquery-ui.min.css` defines `position: relative;` for the `ui-dialog-content` class, and `bootstrap.min.css` defines `left: 50%;` for the `modal-dialog` class, which conflict with each other causing the display issue.

This can be fixed by adding this to the main theme CSS file:

`#crm-container .modal-dialog,.crm-container .modal-dialog,#crm-volunteer-dialog.ui-dialog-content {
	left:auto;
}`

Can this be mitigated in some way in the extension itself? i.e. can the above CSS be added to one of the extension's CSS files when one of these forms is loaded?

**Comment by kelizoliva** [comment]
Right, but up until recently (in my case), CiviVolunteer loaded the admin theme (Seven). Now it is loading the front-end theme. The rest of Civi loads the correct theme. Why is this?

**Comment by ginkgomzd** [comment]
Hi @kelizoliva this sounds like an unrelated issue to me. It doesn't sound like a CiviVolunteer issue at all really. CiviVolunteer does not load a theme though the URL used to access CiviVolunteer might trigger different theme loading behaviour than you expect.

**Comment by kelizoliva** [comment]
Yeah, I needed to aggressively clear caches and reset the asset cache. Disregard my input!
